app/prepare_proposal.go

```diff
@@ -60,7 +60,7 @@ func (app *App) PrepareProposal(req abci.RequestPrepareProposal) abci.ResponsePr
 		txs = FilterTxs(app.Logger(), sdkCtx, handler, app.txConfig, req.BlockData.Txs)
 	}
 
-	// build the square from the set of valid and prioritised transactions.
+	// build the square from the set of valid and prioritized transactions.
```

Comment:
old version seems fine

```suggestion
	// build the square from the set of valid and prioritised transactions.
```

<img width="357" alt="Screenshot 2023-12-16 at 2 51 42 PM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/e9088c3c-9518-4a52-8090-8149db14d6aa">